src/terrain/util/config.clj

```diff
@@ -336,141 +336,140 @@
        (cfg/env-setting "DATA_INFO_PORT")
        (data-info-base-url)))))
 
-(cc/defprop-str tree-parser-url
+(cc/defprop-optstr tree-parser-url
   "The URL for the tree parser service."
   [props config-valid configs tree-viewer-routes-enabled]
-  "terrain.tree-viewer.base-url")
+  "terrain.tree-viewer.base-url" "http://portnoy.iplantcollaborative.org/parseTree")
 
-(cc/defprop-str es-url
+(cc/defprop-optstr es-url
   "The URL for Elastic Search"
   [props config-valid configs data-routes-enabled]
-  "terrain.infosquito.es-url")
+  "terrain.infosquito.es-url" "http://elasticsearch:9200")
 
-(cc/defprop-str jwt-private-signing-key
+(cc/defprop-optstr jwt-private-signing-key
   "The path to the private key used for signing JWT assertions."
   [props config-valid configs]
-  "terrain.jwt.signing-key.private")
+  "terrain.jwt.signing-key.private" "/path/to/private-key.pem")
```

Comment:
True. I was a little ambivalent about this change and others like it because on one hand, it would be nice to have the actual paths in these settings so that we could rely on the defaults, but was reluctant to supply this information to others. I suppose that wouldn't be _too_ risky, though. If someone actually gains access to the files, we have bigger problems anyway.